index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="/styles/reset.css" />
+    <link rel="stylesheet" href="/styles/index.css" />
+    <title>Static Template</title>
+  </head>
+  <body>
+    <div
+      class="three-column-grid__expand-two gap-sm center-wrapper main-layout"
+    >
+      <header class="main-header three-row-grid__expand-two gap-sm">
+        <div class="box-md"></div>
+        <nav>
+          <ul class="nav-list">
+            <li>
+              <a class="nav-list--link" href="#">
+                <div class="frame">
+                  <img src="/assets/home.png" alt="home icon" />
+                </div>
+                <div class="link-text">Home</div>
+              </a>
```

Comment:
I really like the HTML structure. By wrapping the `img` and the `div.link-text` it means when a user clicks either of them it will re-direct them to the relevant page.